app/src/main/java/com/practicum/playlistmaker/AppThemeMode.kt

```diff
@@ -0,0 +1,40 @@
+package com.practicum.playlistmaker
+
+import android.app.Application
+import android.content.SharedPreferences
+import android.content.res.Configuration
+import androidx.appcompat.app.AppCompatDelegate
+
+class AppThemeMode : Application() {
+
+    var darkTheme = false
+    lateinit var sharedPreferences: SharedPreferences
+
+    override fun onCreate() {
+        super.onCreate()
+
+        sharedPreferences = getSharedPreferences("SHARED_PREFERENCE", MODE_PRIVATE)
```

Comment:
```suggestion
        sharedPreferences = getSharedPreferences(SHARED_PREFERENCE, MODE_PRIVATE)
```
Здесь тоже лучше использовать ключ из константы, а не строчку